.github/workflows/autoassign.yml

```diff
@@ -0,0 +1,12 @@
+name: 'Auto Assign PR'
```

Comment:
Rename file to: `platform.autoassign.yml`.

However, maybe we want to be a bit more descriptive and could even name it `platform.auto-assign-PR`

Reply:
Fixed.